CONTRIBUTING.md

```diff
@@ -0,0 +1,30 @@
+## Developer Git Workflow:
+
+- 📌 Clone down the organization's repo as your `origin`
+- 📌 Pick up a "ticket" from our [GitHub KanBan Board](https://github.com/orgs/QueerGlobal/projects/1/views/2) `Todo` lane
```

Comment:
We need to make that project board publicly accessible, because it isn't right now.

Reply:
I thought the same thing but then I saw this for [Settings - Frontend Roadmap](https://github.com/orgs/QueerGlobal/projects/1/settings/access)

![image](https://github.com/QueerGlobal/qg-frontend-v2/assets/92059005/bdc57182-90c1-4d31-9468-3fdc4c6a34ca)

Reply:
Is there a reason it wouldn't be visible to Mark? (or you as well Colin since you had to dig to find the public settings?)